invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/menu.overrides

```diff
@@ -2,25 +2,31 @@
          Site Overrides
 *******************************/
 
-.ui.inverted.menu{
-  background: @brandColor;
+.ui.inverted.menu {
   border-radius: 0px;
 }
 
-.ui.vertical.menu .active.item {
-    background: @primaryColor;
-    pointer-events: none;
-}
-
 .ui.vertical.menu .item i.icon {
     float: left;
     margin-right: 0.5em;
 }
 
-.ui.menu .blue.active.item {
-    color: white !important;
+/* Brand */
+.ui.inverted.menu .brand.active.item,
+.ui.inverted.brand.menu {
+  background-color: @brandColor;
+}
+.ui.inverted.brand.menu .item:before {
+  background-color: @invertedColoredDividerBackground;
+}
+.ui.inverted.brand.menu .active.item {
+  background-color: @invertedColoredActiveBackground !important;
 }
 
-.ui.vertical.menu .active.item:hover {
-    background-color: @primaryColor;
+.ui.menu .brand.active.item,
+.ui.brand.menu .active.item {
+  border-color: @brandColor !important;
+  background-color: @brandColor;
+  color: @white !important;
```

Comment:
@lnielsen do you think we should create a specific variable for that and default to white? I would say that having the active item be always white is fine but let me know if you think it should be also configurable!

Reply:
Let's stay with it for now. It's overwritable by an instance.